modules/gke-cluster-autopilot/variables.tf

```diff
@@ -50,21 +50,14 @@ variable "description" {
 variable "enable_addons" {
   description = "Addons enabled in the cluster (true means enabled)."
   type = object({
-    cloudrun                   = optional(bool, false)
```

Comment:
Why are you removing more attributes? Please provide a rationale for doing that in the PR description, with links to the doc where it says those are not supported.

Reply:
![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/93377317/16e13888-1d92-478c-b618-53cf0f89d372)
- Network policy parameter conflicts with Autopilot cluster (https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy#enabling_network_policy_enforcement doc says it's enable by default)

![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/93377317/a0165d7d-dc06-496c-89fc-8840a5f78d4c)
- DNS Cache conflicts with Autopilot cluster (https://cloud.google.com/kubernetes-engine/docs/how-to/nodelocal-dns-cache doc says it's enabled by default)

![image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/assets/93377317/f6fa1187-5eb3-4011-9811-fc98a103e8f7)

- _horizontal-pod-autoscaling_ and _http-load-balancing_ are required to be enabled, so I enforced it in the code, because if users choose _false_ the apply fails.

Reply:
Awesome, can you drop a comment in the code explaining why each of those options is hardcoded for autopilot? What you just said in your comment would be perfect